data/mp/multiplay/skirmish/nb_includes/_events.js

```diff
@@ -10,19 +10,19 @@ function eventStartLevel() {
 		// initialize subpersonality pseudo-randomly here
 		// to make sure teammates have the same personality
 		var j = 1, s = 0;
-		for (var i = 0; i < maxPlayers; ++i) {
+		for (let i = 0; i < maxPlayers; ++i) {
 			if (allianceExistsBetween(me, i))
 				s += j;
 			j *= 2;
 		}
 		// the random "s" number obtained here is the same for all players in any team
 		var s = s + (new Date()).getMinutes();
 		j = 0;
-		for (var i in subpersonalities) // count the amount of subpersonalities
+		for (const i in subpersonalities) // count the amount of subpersonalities
 			++j;
 		s = s % j;
```

Comment:
should i change this to:
```js
s = s % Object.keys(subpersonalities).length;
```
this should fix the problem that is reported by lgtm-com

Reply:
change is done